adapters.go

```
@@ -28,7 +29,9 @@ func AdaptPayloadTxsToCosmosTxs(ethTxs []hexutil.Bytes) (bfttypes.Txs, error) {
 	if ethL1AttributesTx.Type() != ethtypes.DepositTxType {
 		return nil, errors.New("first transaction is not a deposit transaction")
 	}
-	l1BlockInfo, err := derive.L1BlockInfoFromBytes(&rollup.Config{}, uint64(time.Now().Unix()), ethL1AttributesTx.Data())
+	l1BlockInfo, err := derive.L1BlockInfoFromBytes(&rollup.Config{
+		EcotoneTime: utils.Ptr(uint64(0)), // TODO: this is a hack, but it works for now.
+	}, uint64(time.Now().Unix()), ethL1AttributesTx.Data())
```

golangci-lint (1.62.2)

[high] G115: integer overflow conversion int64 -> uint64

(gosec)